hack/build-manifests.sh

```diff
@@ -372,7 +373,8 @@ ${PROJECT_ROOT}/tools/csv-merger/csv-merger \
   --webhook-image-name="${HCO_WEBHOOK_IMAGE}" \
   --kubevirt-consoleplugin-image-name="${KUBEVIRT_CONSOLE_PLUGIN_IMAGE}" \
   --kubevirt-consoleproxy-image-name="${KUBEVIRT_CONSOLE_PROXY_IMAGE}" \
-  --cli-downloads-image-name="${HCO_DOWNLOADS_IMAGE}" > "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}"
+  --cli-downloads-image-name="${HCO_DOWNLOADS_IMAGE}" > "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}" \
+  --primary-udn-binding-image-name="${PRIMARY_USER_DEFINED_NETWORK_BINDING_IMAGE}"
```

Comment:
`>` to a file should be the after all the params i think

Reply:
good catch! thanks!